MAINTAINING.md

```diff
@@ -0,0 +1,59 @@
+# Maintaining React-Bootstrap
+
+This document is primarily for members of the BlessCSS organization, though
+publicly available for your viewing pleasure. It describes how we work as a
+team to provide our downstream users with a great product. You will find
+descriptions for common tasks such as triaging, merging pull requests, and release
+procedures.
+
+If you are interested in contributing to Bless, you should check out the
+[Contributing Guide](./CONTRIBUTING.md).
+
+## Triaging Issues
+
+New issues pop up, We need to identify urgent issues (such as nobody can use the
```

Comment:
I fixed the language on this section, that was another copy and paste fail on my end.